.github/workflows/build-and-test.yml

```diff
@@ -182,7 +182,7 @@ jobs:
           fi
 
       - name: Run the integration test suite
-        run: ./gradlew --build-cache runTest --args="suites/dev/FullIntegration.json build/reports"
+        run: ./gradlew --build-cache runTest --args="suites/local/FullIntegration.json build/reports"
```

Comment:
It looks like your branch is out of date with `main` in some way. This diff shouldn't be showing up in your PR.